index.html

```diff
@@ -251,7 +303,8 @@ <h3>Skills</h3>
           <button type="submit" class="see-project-button">Get in touch</button>
```

Comment:
## OPTIONAL 
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/154773290-8f0c742b-f4f4-4edf-b345-563421dd33ac.png)  | ![touchh](https://user-images.githubusercontent.com/69638013/154773220-dfeff1b4-cfa5-4420-b534-9fd29c52b50d.PNG)

- [ ] Kindly fix the alignment of this button to look like the original design 👍🏿